environments/cp_4.2.6.yml

```diff
@@ -0,0 +1,22 @@
+
+#modeled off of  "NF1_SchwannCell_data_analysis/environment.yml" & "nf1_cellpaining_data/nf1_cellpainting_env.yml"
```

Comment:
Recommend moving this environment code up one so that it starts at line 1.

Reply:
Done!